utils/run_test.sh

```diff
@@ -98,9 +98,10 @@ if [ -d "/mnt/daos" ]; then
     run_test "${SL_PREFIX}/bin/vea_ut"
     run_test src/rdb/raft_tests/raft_tests.py
     # Satisfy CGO requirements for go-spdk binding and internal daos imports
-    LD_LIBRARY_PATH="${SL_PREFIX}/lib:${SL_SPDK_PREFIX}/lib:${LD_LIBRARY_PATH}"
+    LD_LIBRARY_PATH=\
```

Comment:
(lint) This assignment is only seen by the forked process. [SC2097]
(lint) Remove space after = if trying to assign a value (for empty string, use var='' ... ). [SC1007]

Reply:
Maybe you can't split an assignment like this.